src/o365/teams/commands/tab/tab-remove.ts

```diff
@@ -0,0 +1,145 @@
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import {
+  CommandOption, CommandValidate
+} from '../../../../Command';
+import Utils from '../../../../Utils';
+import { Tab } from '../../Tab';
+import { GraphItemsListCommand } from '../../../base/GraphItemsListCommand';
+import request from '../../../../request';
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  teamId: string;
+  channelId: string;
+  tabId: string;
+  confirm?: boolean;
+}
+class TeamsTabRemoveCommand extends GraphItemsListCommand<Tab> {
```

Comment:
Since this command is not retrieving tabs, it should inherit from the `GraphCommand` base class rather than `GraphItemsListCommand<Tab>`

Reply:
Thanks for catching this , cheers